docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -160,6 +160,14 @@ cqlsh:example> SELECT * FROM devices;
            1 |       2 | Acme Corporation | Giant Rubber Band
 ```
 
+### Use table property to define the number of tablets
+
+You can do this as shown below.
+
+```cassandraql
+cqlsh:example> CREATE TABLE users(user_id INT PRIMARY KEY, full_name TEXT) WITH tablets = 10;
```

Comment:
Add another example using `TABLETS` together with another table option - like:
`WITH tablets = 10 AND transactions = { 'enabled' : true };`